code/Model/SiteTree.php

```diff
@@ -1449,6 +1464,47 @@ public function MetaComponents()
         return $tags;
     }
 
+    /**
+     * Create the value for the meta generator tag
+     * Will suffix on the major.minor version of a stable tag
+     *
+     * @return string
+     */
+    private function getGenerator(): string
+    {
+        $generator = trim(Config::inst()->get(self::class, 'meta_generator'));
+        if ($generator === '') {
+            return '';
+        }
+        if (self::config()->get('show_meta_generator_version')) {
+            $version = $this->getVersionProvider()->getModuleVersion('silverstripe/framework');
```

Comment:
There was a whole exercise of fallbacks within CWP to show something more relevant than just the framework version or the single recipe version, is framework the right source of this information? It's the most reliable one, though it's not uncommon to use a newer framework version than CMS or the recipe.

Reply:
It's an interesting question.

The version number displayed in the CMS is meant for actual CMS users. This one is really meant for robots, so I think it makes less sense to have the convoluted fallback logic.

Since this meta tag is coming from `silverstripe/cms`, maybe it would make more sense to track that module ... but I don't have string feelings either way.

Reply:
At some point in the future we could theoretically expose this number on a non traditional site without pages that doesn't have CMS installed, possibly via an API endpoint (I'm speculating here).  In that scenario we'd get the version from framework, so it makes sense to use framework here so the number would be aligned.